permissions/plugin-portshift-scanner.yml

```
@@ -0,0 +1,10 @@
+---
+name: "portshift-scanner"
+github: "jenkinsci/portshift-ci-plugin"
+paths:
+- "org/jenkins-ci/plugins/portshift-scanner"
```

This should be `io/jenkins/plugins/portshift-ci`, it's the groupId from your pom.xml (https://github.com/jenkinsci/portshift-ci-plugin/blob/master/pom.xml#L10) and the artifactId from the pom.xml (https://github.com/jenkinsci/portshift-ci-plugin/blob/master/pom.xml#L11) with periods replaced with `/`

---

@slide , I've changed the artifact id to portshift-scanner so it supposed to be OK (I opened this PR before I pushed my changes to the portshift ci scanner repo)

---

The path is still wrong, you need to use the correct groupId (io/jenkins/plugins).
```suggestion
- "io/jenkins/plugins/portshift-scanner"
```